server/test/unit/plugins/util/test_publish_step.py

```diff
@@ -116,6 +119,24 @@ def test_get_status_conduit_from_parent(self):
         step.parent.get_status_conduit.return_value = 'foo'
         self.assertEquals('foo', step.get_status_conduit())
 
+    def test_get_total(self):
+        step = Step('foo_step')
+        self.assertEquals(1, step.get_total())
+
+    def test__get_total(self):
```

Comment:
I'm not sure what the point of this test is. A method is set with mock and then called and asserted that mock did its job.